plugin/executor/containerd/containerd.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // PluginNamespace is the name used for the plugins namespace
-var PluginNamespace = "moby-plugins"
+var PluginNamespace = "plugins.moby"
```

Comment:
We'll need a way to migrate users from the old namespace. This is particularly a problem when live-restore is enabled... without live-restore we could just delete the old container and create a new one in the new namespace.

Even with live-restore we can probably get away with shutting down the containers and re-creating them in the new namespace.

Reply:
Yes, we should have a look at migration, but impact is limited currently to people running 17.11 (which is an edge release); it's important we make sure that this change will be in 17.12.0 (worst case scenario: without the migration step, but instructions in the release notes to remove plugins and re-add them after updating)

Reply:
@stevvooe @crosbymichael  is it possible to "change" an object namespace in containerd ?

Reply:
The namespace cannot be renamed.

We should have considered this before merging containerd 1.0 support. We can migrate but the only impact here is that plugins will be impacted on upgrade.

Reply:
As far as the impact goes we can include a note in the changelog with a link to a script that can do the migration.

I think that should suffice since the number of users using 17.11 along with plugins, in production, is probably low.

Reply:
I may have already answered this elsewhere, but no.